src/jquery.multiselect.js

```diff
@@ -19,30 +19,79 @@
 (function($, undefined) {
    // Counter used to prevent collisions
    var multiselectID = 0;
-   // Scroll bar width saved for auto menu width determinations.
-   var _scrollbarWidth = 0;
-
-   var defaultIcons = {
-      'open': '<span class="ui-icon ui-icon-triangle-1-s"></span',
-      'close': '<span class="ui-icon ui-icon-circle-close"></span>',
-      'checkAll': '<span class="ui-icon ui-icon-check"></span>',
-      'uncheckAll': '<span class="ui-icon ui-icon-closethick"></span>',
-      'flipAll': '<span class="ui-icon ui-icon-arrowrefresh-1-w"></span>'
+
+   // The following information can be overridden via the linkInfo option.
+   // An $.extend is used to allow just specifying a partial object in linkInfo.
+   var linkDefaults = {
+      'open': {
+         'class': 'ui-multiselect-open',
+         'icon': '<span class="ui-icon ui-icon-triangle-1-s"></span',
+         'text': 'NOT USED',
+         'title': 'Open'
+      },
+      'close': {
+         'class': 'ui-multiselect-close',
+         'icon': '<span class="ui-icon ui-icon-circle-close"></span>',
+         'text': 'NOT USED',
+         'title': 'Close'
+      },
+      'checkAll': {
+         'class': 'ui-multiselect-all',
+         'icon': '<span class="ui-icon ui-icon-check"></span>',
+         'text': 'Check all',
+         'title': 'Check all'
+      },
+      'uncheckAll': {
+         'class': 'ui-multiselect-none',
+         'icon': '<span class="ui-icon ui-icon-closethick"></span>',
+         'text': 'Uncheck all',
+         'title': 'Uncheck all'
+      },
+      'flipAll': {
+         'class': 'ui-multiselect-flip',
+         'icon': '<span class="ui-icon ui-icon-arrowrefresh-1-w"></span>',
+         'text': 'Flip all',
+         'title': 'Flip all'
+      },
+      'collapse': {
+         'class': 'NOT USED',
+         'icon': '<span class="ui-icon ui-icon-triangle-1-s"></span>',
+         'text': 'NOT USED',
+         'title': 'Collapse'
+      },
+      'expand': {
+         'class': 'NOT USED',
```

Comment:
Do these even need to exist if they're not used?

Reply:
Mainly there for documenting so that people do not think they were missed and try to use them.  I can remove them.

Reply:
@mlh758 So do you want me to remove them then?  
I could put the "NOT USED" bit in comments if that would be better.

Reply:
I think you can just remove them, not really a need for a comment either. It seems like their lack of existence would be enough to indicate they aren't used.

Reply:
@mlh758 ok